src/manager/RestCountriesManager.ts

```diff
@@ -26,14 +26,26 @@ export class RestCountriesManager {
 
     @RunEvery(31, METHOD_EXECUTOR_TIME_UNIT.days, true)
     private async init(): Promise<void> {
+        const englishNotPrimary = "BI,CM,SZ,IN,KI,LS,MT,MH,NA,NR,PK,PW,PH,RW,WS,SC,SD,TO,TV,VU";
```

Comment:
make class level constant

Reply:
also might be better making this an array

Reply:
Made class level and array